store/indicators.js

```diff
@@ -0,0 +1,95 @@
+// this store manages NCAR 12km indicator data
+import _ from 'lodash'
+import { convertMmToInches, convertValueToFahrenheit } from '../utils/convert'
+import { getHttpError } from '../utils/http_errors'
+
+var convertTemperatureData = function (obj) {
+  if (typeof obj === 'number') {
+    return convertValueToFahrenheit(obj)
+  } else if (typeof obj === 'object') {
+    for (let key in obj) {
+      obj[key] = convertTemperatureData(obj[key])
+    }
+    return obj
```

Comment:
What happens if the object type isn't number or object?  Should be an error state, ...?  see above